modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/data_renderers/ActionsDropdownRenderer.js

```diff
@@ -176,15 +176,14 @@ function ActionsDropdownRenderer({actions, itemData, itemId}) {
 		return null;
 	}
 
-	if (actions.length === 1) {
-		const action = formattedActions[0];
+	if (formattedActions.length === 1) {
+		const [action] = formattedActions;
+		const {data: actionData} = action;
 
-		if (action.id && !action.href) {
+		if (actionData?.id && !actionData?.href) {
```

Comment:
Perhaps overly defensive, but optional check makes sure the code runs smoothly if `data` is `undefined`.